Makefile

```diff
@@ -112,6 +112,9 @@ test-unit:
 	@go test $(TEST_OPTIONS) $(PKGS)
 test-e2e:
 	hack/test-e2e.sh
+test-sec:
+	@which gosec 2> /dev/null >&1 || { echo "gosec must be installed to lint code";  exit 1; }
```

Comment:
You could `go get -u github.com/path/to/gosec` - we do that with `imagebuilder`

Reply:
done!